mapping.json

```diff
@@ -1,38 +1,48 @@
-        ["google-analytics.com/inpage_linkid.js", "inpage_linkid.js"],
-        ["google-analytics.com/plugins/ga/inpage_linkid.js", "inpage_linkid.js"],
-        ["google-analytics.com/cx/api.js", "api.js"],
-        ["google-analytics.com/gtm/js", "gtm.js"]
+        { "url": "google-analytics.com/ga.js", "surrogate": "ga.js" },
+        { "url": "google-analytics.com/analytics.js", "surrogate": "analytics.js" },
+        { "url": "google-analytics.com/inpage_linkid.js", "surrogate": "inpage_linkid.js" },
+        { "url": "google-analytics.com/plugins/ga/inpage_linkid.js", "surrogate": "inpage_linkid.js" },
+        { "url": "google-analytics.com/cx/api.js", "surrogate": "api.js" },
+        { "url": "google-analytics.com/gtm/js", "surrogate": "gtm.js" }
     ],
     "googletagmanager.com": [
-        ["googletagmanager.com/gtm.js", "gtm.js"]
+        { "url": "googletagmanager.com/gtm.js", "surrogate": "gtm.js" }
     ],
     "googletagservices.com": [
-        ["googletagservices.com/gpt.js", "gpt.js"],
-        ["googletagservices.com/tag/js/gpt.js", "gpt.js"]
-    ],        
+        { "url": "googletagservices.com/gpt.js", "surrogate": "gpt.js" },
+        { "url": "googletagservices.com/tag/js/gpt.js", "surrogate": "gpt.js" }
+    ],
     "googlesyndication.com": [
-        ["googlesyndication.com/adsbygoogle.js", "adsbygoogle.js"],
-        ["googlesyndication.com/pagead/js/adsbygoogle.js", "adsbygoogle.js"]
+        { "url": "googlesyndication.com/adsbygoogle.js", "surrogate": "adsbygoogle.js" },
+        { "url": "googlesyndication.com/pagead/js/adsbygoogle.js", "surrogate": "adsbygoogle.js" }
     ],
     "doubleclick.net": [
-        ["doubleclick.net/instream/ad_status.js", "ad_status.js"],
-        ["doubleclick.net/tag/js/gpt.js", "gpt.js"]
+        { "url": "doubleclick.net/instream/ad_status.js", "surrogate": "ad_status.js" },
+        { "url": "doubleclick.net/tag/js/gpt.js", "surrogate": "gpt.js" }
     ],
     "scorecardresearch.com": [
-        ["scorecardresearch.com/beacon.js", "beacon.js"]
+        { "url": "scorecardresearch.com/beacon.js", "surrogate": "beacon.js" }
     ],
     "outbrain.com": [
-        ["outbrain.com/outbrain.js", "outbrain.js"]
+        { "url": "outbrain.com/outbrain.js", "surrogate": "outbrain.js" }
     ],
     "amazon-adsystem.com": [
-        ["amazon-adsystem.com/aax2/amzn_ads.js", "amzn_ads.js"]
+        { "url": "amazon-adsystem.com/aax2/amzn_ads.js", "surrogate": "amzn_ads.js" }
     ],
     "chartbeat.com": [
-        ["chartbeat.com/chartbeat.js", "chartbeat.js"],
-        ["chartbeat.com/js/chartbeat.js", "chartbeat.js"]
+        { "url": "chartbeat.com/chartbeat.js", "surrogate": "chartbeat.js" },
+        { "url": "chartbeat.com/js/chartbeat.js", "surrogate": "chartbeat.js" }
+    ],
+    "facebook.net": [
+        { "regexUrl": "facebook\\.net\\/[a-z_A-Z]+\\/sdk\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" },
```

Comment:
sorry, I missed this discussion - why would putting those regexes into "url" not work? Or it would work but we are creating `regexUrl` so that we can do the regex-specific validation in `test-mapping.js`?

Reply:
We discussed this around here https://app.asana.com/0/0/1202673425163833/1202706967057780/f, but the gist of it is that when building `mapping.js`, the URL is escaped to build a valid regex.  If its already a regex, it would end up being double-escaped.  To avoid that (and some hacky "is it regex or not" auto-detection) we decided on having a separate property to denote a regex URL.

Reply:
Gotcha, that makes sense to me although I wonder why just not convert all `url`s to `regexUrl`? Do we need `url` if we have `regexUrl`?

Reply:
@kdzwinel sure, we can just replace all of the `url` properties with `regexUrl` instead in `mapping.js`